ci-operator/step-registry/rosa/cluster/upgrade/machinepool/rosa-cluster-upgrade-machinepool-commands.sh

```
@@ -0,0 +1,103 @@
+fi
+
+UPGRADED_TO_VERSION=${UPGRADED_TO_VERSION:-}
+if [[ -z "$UPGRADED_TO_VERSION" ]]; then
+  UPGRADED_TO_VERSION=$(rosa describe cluster -c $cluster_id -o json | jq -r '.openshift_version')
+  log "No UPGRADED_TO_VERSION specified, using current control plane version: $UPGRADED_TO_VERSION"
+fi
+
+log "rosa list machinepool -c $cluster_id"
+rosa list machinepool -c $cluster_id
+
+mp_id_list=$(rosa list machinepool -c $cluster_id -o json | jq -r ".[].id")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)